src/CAREUI/interactive/WeekdayCheckbox.tsx

```diff
@@ -29,49 +30,50 @@ interface Props {
   format?: "alphabet" | "short" | "long";
 }
 
-export default function WeekdayCheckbox({
-  value = [],
-  onChange,
-  format = "alphabet",
-}: Props) {
-  const selectedDays = value ?? [];
-  const { t } = useTranslation();
+const WeekdayCheckbox = forwardRef<HTMLDivElement, Props>(
```

Comment:
https://react.dev/reference/react/forwardRef

pass ref as a prop instead, forwardRef is being deprecated.